ui/labels.js

```diff
@@ -0,0 +1,63 @@
+/*!
+ * jQuery UI Labels @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: labels
+//>>group: Core
+//>>description: Find all the labels associated with a given input
+//>>docs: http://api.jqueryui.com/category/labels/
```

Comment:
Should be "http://api.jqueryui.com/labels/", also need to file a ticket to create that page.